x-pack/solutions/security/plugins/security_solution/public/common/components/visualization_actions/types.ts

```diff
@@ -43,6 +43,7 @@ export interface UseLensAttributesProps {
   stackByField?: string;
   title?: string;
   esql?: string;
+  signalIndexName?: string | null;
```

Comment:
I know the rest of the props don't have it, but could you write some quick doc here to indicate what's the new prop is doing?

Same for the other 2 places below in this file.